src/zenml/integrations/gcp/flavors/vertex_experiment_tracker_flavor.py

```diff
@@ -0,0 +1,180 @@
+
+        Returns:
+            The experiment.
+        """
+        if value and not re.match(r"^[a-z0-9][a-z0-9-]{0,127}$", value):
+            raise ValueError(
+                "Experiment name must match regex [a-z0-9][a-z0-9-]{0,127}"
+            )
+        return value
+
+
+class VertexExperimentTrackerConfig(
+    BaseExperimentTrackerConfig,
+    GoogleCredentialsConfigMixin,
+    VertexExperimentTrackerSettings,
+):
+    """Config for the VertexAI experiment tracker.
+
+    Attributes:
+        location: Optional. The default location to use when making API calls. If not
+            set defaults to us-central-1.
+        staging_bucket: Optional. The default staging bucket to use to stage artifacts
+            when making API calls. In the form gs://...
+        network:
+            Optional. The full name of the Compute Engine network to which jobs
+            and resources should be peered. E.g. "projects/12345/global/networks/myVPC".
+            Private services access must already be configured for the network.
+            If specified, all eligible jobs and resources created will be peered
+            with this VPC.
+        service_account:
+            Optional. The service account used to launch jobs and deploy models.
+            Jobs that use service_account: BatchPredictionJob, CustomJob,
+            PipelineJob, HyperparameterTuningJob, CustomTrainingJob,
+            CustomPythonPackageTrainingJob, CustomContainerTrainingJob,
+            ModelEvaluationJob.
+        encryption_spec_key_name:
+            Optional. The Cloud KMS resource identifier of the customer
+            managed encryption key used to protect a resource. Has the
+            form:
+            ``projects/my-project/locations/my-region/keyRings/my-kr/cryptoKeys/my-key``.
+            The key needs to be in the same region as where the compute
+            resource is created.
+        api_endpoint (str):
+                Optional. The desired API endpoint,
+                e.g., us-central1-aiplatform.googleapis.com
+        api_key (str):
+            Optional. The API key to use for service calls.
+            NOTE: Not all services support API keys.
+        api_transport (str):
+            Optional. The transport method which is either 'grpc' or 'rest'.
+            NOTE: "rest" transport functionality is currently in a
+            beta state (preview).
+        request_metadata:
+            Optional. Additional gRPC metadata to send with every client request.
+    """
+    location: Optional[str] = None
+    staging_bucket: Optional[str] = None
+    network: Optional[str] = None
+    service_account: Optional[str] = None
+    encryption_spec_key_name: Optional[str] = None
+    api_endpoint: Optional[str] = None
+    api_key: Optional[str] = None
+    api_transport: Optional[str] = None
+    request_metadata: Optional[dict] = None
+
+
+class VertexExperimentTrackerFlavor(BaseExperimentTrackerFlavor):
```

Comment:
I think we should test whether we can use service connectors to authenticate this experiment tracker. To implement this you'll actually only need to specify the `service_connector_requirements` property on the flavor here (see e.g. the vertex orchestrator flavor for an example)

Reply:
I’ve added service_connector_requirements to the flavor as you suggested, but when running a command like:
`zenml experiment-tracker connect <MY_COMPONENT_NAME> -i`
I’m getting:

> Error: The <MY_COMPONENT_NAME> experiment tracker implementation does not support using a service connector to connect to resources.

Am I missing something?